debian/control

```diff
@@ -25,6 +25,8 @@ Architecture: all
 Depends: ${shlibs:Depends},
          ${misc:Depends},
          gnome-shell-communitheme (= ${source:Version}),
+         gtk-communitheme,
```

Comment:
I've been wondering; what does `(= ${source:Version})` do exactly and why is it only added for `gnome-shell-communitheme`. :thinking: 

Also, do you know why the build throws a warning on `${shlibs:Depends}` and `${misc:Depends}` (and what does this mean? :smile:)


https://launchpadlibrarian.net/351425683/buildlog_ubuntu-artful-amd64.gnome-shell-communitheme_18.04~wip1+201712282046+e69fb5a~ubuntu17.10.1_BUILDING.txt.gz

```
dpkg-gencontrol: warning: Depends field of package ubuntu-communitheme-session: unknown substitution variable ${shlibs:Depends}
dpkg-gencontrol: warning: Depends field of package gnome-shell-communitheme: unknown substitution variable ${shlibs:Depends}
```

Reply:
As the 2 binary packages are from the same source package, we force to have synced up version (and this is important as the session file has the gnome-shell name reference). We can't do the same with other packages, as they are from different source.

On the warning, this is normal as we don't compile anything, so `dh_shlibdeps` isn't called and the variable not substituted. You will notice that `${misc:Depends}` doesn't trigger a warning, and it's replaced dynamically by `dconf-gsettings-backend | gsettings-backend` for instance, on the -session package.
Those are basically variables resolving packaging deps that can be detected at build time automatically. (the `dh_*` utilities handle this)